x11-libs/libxkbcommon/libxkbcommon-1.3.0.ebuild

```diff
@@ -55,15 +55,3 @@ multilib_src_configure() {
 	)
 	meson_src_configure
 }
-
-multilib_src_compile() {
-	meson_src_compile
-}
-
-multilib_src_test() {
-	virtx meson_src_test
```

Comment:
sure?

Reply:
Oops. I didn't catch this because the only one of the tests needs an Xserver and it nicely skips if it doesn't have one.